app/scripts/controllers/transactions/index.js

```diff
@@ -366,7 +366,42 @@ class TransactionController extends EventEmitter {
     this.txStateManager.setTxStatusSubmitted(txId)
   }
 
-  confirmTransaction (txId) {
+  /**
+    Sets the status of the transaction to confirmed
```

Comment:
JSDoc comments should be formatter with each line starting with a asterisk (e.g. http://usejsdoc.org/tags-param.html)